Marlin/ultralcd_impl_DOGM.h

```diff
@@ -357,9 +357,10 @@ static void lcd_implementation_init() {
   #endif
 
   #if ENABLED(MKS_12864OLED) || ENABLED(MKS_12864OLED_SSD1306)
-    OUT_WRITE(LCD_PINS_RS, LOW);
-    _delay_ms(500);
-    OUT_WRITE(LCD_PINS_RS, HIGH);
+    #ifndef LCD_RESET
+      #define LCD_RESET
+      #define LCD_RESET_PIN LCD_PINS_RS
+    #endif
```

Comment:
- This file isn't really a proper header, so the `#ifndef` wrapper isn't needed here.
- Should still use `#ifndef LCD_RESET_PIN` so the pin can be overridden elsewhere (e.g., configs).

Reply:
Oh! Just learned that PIN_EXIST(PN) macro is doing the trick! Thanks for fixing my messy code @thinkyhead